run_all_experiments.py

```diff
@@ -63,6 +77,28 @@ def get_experiment_configs(
     print(f'A benchmark yaml file ({args.benchmark_yaml}) is provided. '
           f'Will use it and ignore the files in {args.benchmarks_directory}.')
     benchmark_yamls = [args.benchmark_yaml]
+  elif args.generate_benchmarks:
+    logging.info('Generating benchmarks.')
+    outdir = get_next_generated_benchmarks_dir()
+    logging.info('Setting benchmark directory to %s.', outdir)
+    os.makedirs(outdir, exist_ok=True)
```

Comment:
Nit: I thought `get_next_generated_benchmarks_dir()` intends to ensure the `outdir` will not exist at this point?

Reply:
Correct, I think my thought pattern was "doesn't hurt to allow exist_ok` but removed it now